apps/app/src/components/QuestionItem/QuestionContent.tsx

```diff
@@ -0,0 +1,17 @@
+"use client";
+
+import { MDXRemote, MDXRemoteSerializeResult } from "next-mdx-remote";
+
+import "prismjs/themes/prism-coy.min.css";
+
+type QuestionContentProps = Readonly<{
+	source: MDXRemoteSerializeResult;
+}>;
+
+export const QuestionContent = ({ source }: QuestionContentProps) => {
+	return (
+		<h3 className="prism-coy grow">
```

Comment:
Teraz to już nie powinno być `h3` raczej bo może w środku zawierać kod a nawet nagłówki.